tools/GenerateDocFxStructure/CommandLineOptions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CommandLine;
+
+namespace GenerateDocFxStructure
+{
+    /// <summary>
+    /// Class for command line options.
+    /// </summary>
+    public class CommandlineOptions
+    {
+        /// <summary>
+        /// Gets or sets the destination folder.
+        /// </summary>
+        [Option('d', "destination", Required = true, HelpText = "Folder containing the destination folder..")]
+        public string? DestinationFolder { get; set; }
```

Comment:
if this is not expected to be null do not make it nullable, instead use `= null!;` to suppress the warning